config/routes.rb

```diff
@@ -3,4 +3,10 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  resources :users do
+    resources :posts
+
+  end
+ 
+  root "users#index"
```

Comment:
- [ ] You were supposed to do not expose unnecessary routes, kindly fix your routes as follow:
```suggestion
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

Reply:
Thank you very much